### PR TITLE
Update skew change in v1.29 from n-1 to n-3

### PR DESCRIPTION
**Pull request by pacoxu**
- preflight check KubeletVersion behavior change

Only change the comments here.

Leave the remove of the KubeletVersion from ignorePreflightErrors to v1.32 when all skew CI can remove it.

### CODE REVIEW

File: kinder/ci/kubeadm-periodic.tests.md [hunk 1]
```diff
@@ -71,10 +71,11 @@ Workflow file names: [`skew-[x]-on-[y]`](./workflows)
...
 
-Note that for the time being kubeadm version X does not support skew against a kubelet version X-2,
-similarly to how kubeadm does not support X-2 skew with the control plane. This requires skipping
-the `KubeletVersion` preflight check. In the future if these X-2 tests are no longer possible with kubeadm
-they would have to be adapted on the kinder side or dropped.
+Before kubeadm v1.29, the maximum kubelet skew from the kubeadm version was X-1. This required skipping
+the KubeletVersion preflight check.
+
+After kubeadm v1.29, the maximum kubelet skew from the kubeadm version has become X-3 and KubeletVersion
+preflight check no longer has to be skipped.
```
Comment (neolit123) [inline kinder/ci/kubeadm-periodic.tests.md:78]:
```suggestion
Before kubeadm v1.29, the maximum kubeadm / kubelet skew was X-1. For example, kubeadm v1.28 could deploy a kubelet that is v1.28 or v1.27. After v1.29 the maximum skew has become X-3. For example, kubeadm v1.29 can deploy a kubelet that is as old as v1.26.
```

**Review by neolit123 — COMMENTED** [review]
reworded the comment a little.
please check if you agree and if it's correct.

we need to update this k/website page as well.
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#kubeadm-s-skew-against-the-kubelet

File: kinder/ci/kubeadm-periodic.tests.md (hunk 1 above)
Comment (pacoxu) [reply kinder/ci/kubeadm-periodic.tests.md:78]: So here `deploy` means `init/upgrade/join` operations. This is clear. Updated.

Comment (neolit123) [reply kinder/ci/kubeadm-periodic.tests.md:78]: > So here `deploy` means `init/upgrade/join` operations. This is clear. Updated.

i don't see a new commit?

**Comment by pacoxu** [comment]
> we need to update this k/website page as well.
> https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#kubeadm-s-skew-against-the-kubelet
> (dev-1.29 branch)

Opened https://github.com/kubernetes/website/pull/43769

**Comment by pacoxu** [comment]
I mistakenly created a new branch in kubeadm. 

I recreate a new PR in https://github.com/kubernetes/kubeadm/pull/2953

**Comment by pacoxu** [comment]
Could you help to remove two branches in the repo with `paco` prefix? 

https://github.com/kubernetes/kubeadm/branches/all

**Comment by neolit123** [comment]
> Could you help to remove two branches in the repo with `paco` prefix?
> 
> https://github.com/kubernetes/kubeadm/branches/all

i will clean them

**Pull request closed without merging**